packages/react-virtual/__tests__/index.test.tsx

```diff
@@ -128,12 +130,12 @@ test('should render given dynamic size after scroll', () => {
     target: { scrollTop: 400 },
   })
 
-  expect(screen.queryByText('Row 3')).not.toBeInTheDocument()
-  expect(screen.queryByText('Row 4')).toBeInTheDocument()
-  expect(screen.queryByText('Row 9')).toBeInTheDocument()
-  expect(screen.queryByText('Row 10')).not.toBeInTheDocument()
+  expect(screen.queryByText('Row 2')).not.toBeInTheDocument()
+  expect(screen.queryByText('Row 3')).toBeInTheDocument()
+  expect(screen.queryByText('Row 6')).toBeInTheDocument()
+  expect(screen.queryByText('Row 7')).not.toBeInTheDocument()
 
-  expect(renderer).toHaveBeenCalledTimes(2)
+  expect(renderer).toHaveBeenCalledTimes(3)
```

Comment:
I am not familiar with these tests but curious why it's rendering one more time than before? I'd think it would render less.

Reply:
As in previous test range was calculated only one time when scroll jumped, now we do it again as in dynamic case changing sizes should trigger checking the range.